recipes/sdbus-cpp/all/conanfile.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
```

Comment:
```suggestion

required_conan_version = ">=1.32.0"

```

validate() method is 1.32